themes/ddbasic/template.node.php

```diff
@@ -0,0 +1,369 @@
+ */
+function ddbasic_preprocess_node(&$variables, $hook) {
+  //
+  // Add tpl suggestions for node view modes.
+  if (isset($variables['view_mode'])) {
+    $variables['theme_hook_suggestions'][] = 'node__view_mode__' . $variables['view_mode'];
+    $variables['theme_hook_suggestions'][] = 'node__' . $variables['type'] . '__view_mode__' . $variables['view_mode'];
+  }
+
+  //
+  // Call our own custom preprocess functions.
+  $function = 'ddbasic_preprocess__node__' . $variables['type'];
+  if (function_exists($function)) {
+    call_user_func_array($function, array(&$variables));
+  }
+
+  // Opening hours on library list. but not on the search page.
+  $path = drupal_get_path_alias();
+  if (!(strpos($path, 'search', 0) === 0)) {
+    $hooks = theme_get_registry(FALSE);
+    if (isset($hooks['opening_hours_week']) && $variables['type'] == 'ding_library') {
+      $variables['opening_hours'] = theme('ding_ddbasic_opening_hours_week', array('node' => $variables['node']));
+    }
+  }
+
+  // Add event node specific ddbasic variables.
+  if (isset($variables['content']['#bundle']) && $variables['content']['#bundle'] == 'ding_event') {
+
+    // Add event location variables.
+    if (!empty($variables['content']['field_ding_event_location'][0]['#address']['name_line'])) {
+      $variables['ddbasic_event_location'] = $variables['content']['field_ding_event_location'][0]['#address']['name_line'] . '<br/>' . $variables['content']['field_ding_event_location'][0]['#address']['thoroughfare'] . ', ' . $variables['content']['field_ding_event_location'][0]['#address']['locality'];
+    }
+    else {
+      // User OG group ref to link back to library.
+      if (isset($variables['content']['og_group_ref'])) {
+        $variables['ddbasic_event_location'] = $variables['content']['og_group_ref'];
+      }
+    }
+  }
+
+  // Add tpl suggestions for node view modes.
+  if (isset($variables['view_mode'])) {
+    $variables['theme_hook_suggestions'][] = 'node__view_mode__' . $variables['view_mode'];
+  }
+
+  // For search result view mode move title into left col. group.
+  if (isset($variables['content']['group_right_col_search'])) {
+    $variables['content']['group_right_col_search']['title'] = array(
+      '#theme' => 'link',
+      '#text' => decode_entities($variables['title']),
+      '#path' => 'node/' . $variables['nid'],
+      '#options' => array(
+        'attributes' => array(
+          'title' => $variables['title'],
+        ),
+        'html' => FALSE,
+      ),
+      '#prefix' => '<h2>',
+      '#suffix' => '</h2>',
+    );
+  }
+
+  // Add updated to variables.
+  $variables['ddbasic_updated'] = t('!datetime', array(
+    '!datetime' => format_date(
+      $variables['node']->changed,
+      $type = 'long',
+      $format = '',
+      $timezone = NULL,
+      $langcode = NULL
+    ),
+  ));
+
+  // Modified submitted variable.
+  if ($variables['display_submitted']) {
+    $variables['submitted'] = t('!datetime', array(
```

Comment:
Why is this passed through `t()`? By using the same string as above translations will be the same.